lib/route-handlers.js

```diff
@@ -25,10 +25,15 @@ const cachedJwks = {};
 
 /**
```

Comment:
As an aside, we can go ahead and remove the dependencies (like jws, etc) that we no longer need, and also remove them from package.json

Reply:
Will do.

Reply:
Use eslint to find statements that can be removed from these files.

Reply:
Ran eslint and removed unwanted declarations and statements.